src/main/java/mate/academy/spring/config/WebConfig.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.spring.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@EnableWebMvc
+@Configuration
+@ComponentScan(basePackages = "mate.academy.spring.controller")
+public class WebConfig implements WebMvcConfigurer {
+    @Bean
+    public InternalResourceViewResolver resolver() {
```

Comment:
Don’t push jsp file, jsp dependencies, hello controller, and InternalResourceViewResolver resolver() bean to PR. (Checklist)